clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -3680,6 +3685,72 @@ public void testRemoveMembersFromGroupRetriableErrors() throws Exception {
         }
     }
 
+    @Test
+    public void testRemoveMembersFromGroupRetriableErrorsInMemberResponse() throws Exception {
+        // Retriable errors should be retried
+        String groupId = "instance-1";
+
+        try (AdminClientUnitTestEnv env = new AdminClientUnitTestEnv(mockCluster(1, 0))) {
+            env.kafkaClient().setNodeApiVersions(NodeApiVersions.create());
+
+            env.kafkaClient().prepareResponse(
+                prepareFindCoordinatorResponse(Errors.NONE, env.cluster().controller()));
+
+            MemberResponse memberResponse = new MemberResponse()
+                .setGroupInstanceId(groupId)
+                .setErrorCode(Errors.COORDINATOR_LOAD_IN_PROGRESS.code());
```

Comment:
I guess that we could remove this test as it is not possible to have `COORDINATOR_LOAD_IN_PROGRESS` as an error for a member, right?

Reply:
Right! Removed. Thanks.